### ISSUE TITLE
Shipping fees not recalculating

### ISSUE THREAD

**Issue by npdemers** [wp-e-commerce/WP-e-Commerce#1684]
I recently upgraded my site from WP e-commerce 3.8.12.1 to 38.14.4, and I ran into a couple problems. One of those problems went away when I tried 3.9, but the other persists:

I'm using the WP E-Commerce Weight & Destination Shipping Modules to get a shipping fee based on the shipping continent. In 3.8.12.1 on the checkout page, when I clicked on the "Shipping Address same as Billing Address?" checkbox, the shipping fees were recalculated immediately. This does not happen in 3.8.14.4 or 3.9. I made no changes to the configuration after upgrading.

I'm running WP 4.0, with a custom theme.

**Comment by ncklln** [comment]
Hi - did you ever get to the bottom of this? I have the same issue. http://www.pussyhomeboutique.co.uk
The shipping isn't added until the customer leaves the site and arrives at sagepay. Have tried with the Default Shipping Country script, but makes no difference. Once you click 'update' on the checkout page it works fine...
Thanks in advance

**Comment by npdemers** [comment]
I have, but it's not a solution I'm happy with. It works, but basically I had to add some logic to my theme. Logic that was there in 3.8.12 but missing in 3.9.3, namely function wpsc_update_shipping_quotes() (used to be in wpsc-core/js/wp-e-commerce.js but is now in wpsc-core/js/wpsc-deprecated.js) and the jQuery fired by the "Shipping address same as billing" checkbox.

(Also I tweaked the wpsc-shopping_cart_page.php template a little to keep the Submit button from disappearing when the "Shipping address same as billing" checkbox is clicked.)

**Comment by ncklln** [comment]
Yep I just looked at your site and you've got the same issue as me. i only ship to one country so I want the shipping to calculate automatically on the checkout page - LIKE IT USED TO!! Don't you just hate it when people 'improve' their products and ruin them!
I'm a self-taught website maker so your tweaks look a bit to complicated for me, but thanks for your help anyway. Nick
